src/test/unittest/ut.c

```diff
@@ -40,6 +40,7 @@
  */
 
 #include "unittest.h"
+#include "util.h"
 
```

Comment:
Will fix copyright. Interestingly enough the make cstyle came out clean, however I now notice this when cstyle is run on unittest:

make[1]: Entering directory `/home/sjelinek/ws/nvml/src/test/unittest'
pP *.[ch]
/bin/sh: 1: pP: not found
make[1]: [cstyle] Error 127 (ignored)

Which is why I didn't see the failure and presumably others have the same issue? And my build succeeded on travis which does cstyle, correct? I will fix the cstyle  errors.